bsp/esp-box-3/esp-box-3.c

```diff
@@ -33,6 +33,26 @@ static const char *TAG = "ESP-BOX-3";
 _Static_assert(CONFIG_ESP_LCD_TOUCH_MAX_BUTTONS > 0, "Touch buttons must be supported for this BSP");
 /** @endcond */
 
+static const ili9341_lcd_init_cmd_t vendor_specific_init[] = {
```

Comment:
Could you please describe this change more? Will we get better display colors with this settings?

Reply:
Yes, the new setting will get better colors.
Previously, the VCOM Register provided by the manufacturer were not correct, causing streaks on the screen, We always thought it was a screen issue.
After communicating and verifying with other manufacturers, we found that the VOM register mentioned above was unreasonable, so we need update it.